packages/zero-schema/src/permissions.ts

```diff
@@ -19,6 +19,10 @@ import {clientToServer, NameMapper} from './name-mapper.ts';
 export const ANYONE_CAN = [
   (_: unknown, eb: ExpressionBuilder<Schema, never>) => eb.and(),
 ];
+
+/**
+ * @deprecated Use {@link ANYONE_CAN} instead.
```

Comment:
I'm confused by this one. Where is `ANYONE_CAN` supposed to be used?

Reply:
I guess this is more of a timing question - are CRUD mutators and reads getting deprecated at the same time? If read permissions are still around for a bit, then this could be used there. The current docs change to deprecate CRUD mutators keeps read permissions around for now. cc @aboodman